src/main/java/org/globex/retail/store/catalog/service/CatalogService.java

```diff
@@ -100,4 +101,11 @@ private List<ProductDto> toDto(List<Product> productList) {
         return productList.stream().map(ProductMapper::toDto).collect(Collectors.toList());
     }
 
+    
+    public List<Category> getCategoryList() {
+        List<Category> categoryList = Category.listAll();
+        
+        return categoryList;
+    }
+
```

Comment:
you could simplify this to `return Category.listAll();`

Reply:
Done